src/main/kotlin/world/anhgelus/msmp/alteredrespawn/api/Stack.kt

```diff
@@ -101,13 +108,47 @@ object Stack {
                         respawned.sendTitle("respawned !", null, 0, 20, 0)
                         mplayer.toDefaultCondition()
                         respawned.world.strikeLightningEffect(event.rightClicked.location)
+                        respawned.addPotionEffect(PotionEffectType.BLINDNESS.createEffect(80, 5))
+                        respawned.addPotionEffect(PotionEffectType.SLOW.createEffect(160, 10))
+                        respawned.addPotionEffect(PotionEffectType.WEAKNESS.createEffect(160, 5))
+                        respawned.addPotionEffect(PotionEffectType.CONFUSION.createEffect(160, 1))
+                        this.setTier(respawned, helmet.color.asARGB())
```

Comment:
I think we can create a config to modify it easily.

Reply:
For ? That when player died